suite-native/navigation/src/components/NavigationContainerWithAnalytics.tsx

```diff
@@ -53,11 +53,15 @@ export const NavigationContainerWithAnalytics = ({ children }: { children: React
     }, [colors, isDarkColor]);
 
     const handleNavigationReady = () => {
+        if (!navigationContainerRef.isReady()) return;
```

Comment:
I would be very surprised if this is ever fulfilled, as it's passed as `onReady` handler in the JSX 🤣